File: BECore/app/model/ProjectAcceptanceManage.js
/// <summary>
/// Model EXTJS ProjectAcceptanceManage.
/// </summary>
Ext.define('QLDT.model.ProjectAcceptanceManage', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectAcceptanceManageID',
    idType: 'Guid',
    fields: [
		{ name: "ProjectID", type: "string" },
		{ name: "EstablishedDate", type: "date" },
		{ name: "MeetingDate", type: "date" },
		{ name: "Status", type: "int" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

File: MT.Model/Business/ProjectAcceptanceManage.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
	#region ProjectAcceptanceManage:EntityBase
	/// <summary>
	/// This object represents the properties and methods of a ProjectAcceptanceManage:EntityBase.
	/// </summary>
	public class ProjectAcceptanceManage:EntityBase
	{
        [Key]
		public Guid ProjectAcceptanceManageID {get;set;}
		public Guid ProjectID{get;set;}
		public DateTime EstablishedDate{get;set;}
		public DateTime MeetingDate{get;set;}
		public int Status{get;set;}
		public bool Inactive{get;set;}
		public int SortOrder{get;set;}

        /// <summary>
        /// Danh sách file đính kèm
        /// </summary>
        /// Create by: manh:11.02.2018
        public List<ProjectAcceptanceManage_AttachDetail> Details { get; set; }

    }
	#endregion
}

